README.md

```diff
@@ -21,6 +21,13 @@ Log in:
 sdr --service-url http://sdr-api-server:3000 login
 ```
 
+Register a new object:
```

Comment:
Worth adding some text to the README to make it clearer what distinguishes deposit from register?

Reply:
Sure, added.